### PR TITLE
[iOS] Support RTL

### PR DESCRIPTION
**Pull request by alexey1312**
From discussions https://github.com/RedMadRobot/figma-export/discussions/133

### CODE REVIEW

File: Sources/FigmaExport/Loaders/ImagesLoader.swift [hunk 1]
```diff
@@ -394,4 +395,20 @@ extension Component {
...
+        let hasNotKeywords = keywords.allSatisfy { !description.contains($0) }
+        if hasNotKeywords { return false }        
+        
+        if (description.contains("RTL") ||
+            description.contains("rtl") ||
+            description.contains("Rtl"))  {
+            return true
+        }
+        
+        return false
```
Comment (subdan) [inline Sources/FigmaExport/Loaders/ImagesLoader.swift:412]: Use `localizedCaseInsensitiveContains` method to perform case-insensitive search.

```suggestion
        return description.localizedCaseInsensitiveContains("rtl")
```

**Review by subdan — CHANGES_REQUESTED** [review]
Add info about supporting RTL to the README.md.

**Comment by subdan** [comment]
Hi, @alexey1312. Thanks for the PR! Will you use this feature in your project?

**Comment by alexey1312** [comment]
> Hi, @alexey1312. Thanks for the PR! Will you use this feature in your project?

Hi! Yes, we will 🚀
We know about the [problem](https://useyourloaf.com/blog/flipping-and-localizing-image-assets/), but I tested the solution above, it works well.

Maybe in the future, it's worth adding Stencil Templates, with .flipsForRightToLeftLayoutDirection(true) for icons.

File: README.md [hunk 2]
```diff
@@ -451,6 +452,8 @@ Custom Stencil templates must have the following names:
 
 If a color, icon or image is unique for iOS or Android platform, it should contains "ios" or "android" word in the description field in the properties. If a color, icon or image is used only by the designer, and it should not be exported, the word "none" should be specified in the description field.
 
+If a icon is support RTL (iOS only), it should contains "rtl" word in the description field in the properties.
```
Comment (subdan) [inline README.md:455]:
```suggestion
If an icon supports RTL (iOS only), it should contains "rtl" word in the description field in the properties.

**Pull request merged**